src/gsi/set_crtm_aerosolmod.f90

```diff
@@ -26,10 +27,196 @@ module set_crtm_aerosolmod
+
+    implicit none
+
+!Aerosol types  (CRTM CMAQ LUTs):
+!1. Dust
+!2. Soot
+!3. Water soluble
+!4. Sulfate
+!5. Sea salt
+!6. Water
+!7. Insoluble
+!8. dust-like
+
+    INTEGER(i_kind),  PARAMETER ::        INVALID_AEROSOL = 0
+    INTEGER(i_kind),  PARAMETER ::      DUST_CMAQ_AEROSOL = 1
+    INTEGER(i_kind),  PARAMETER ::           SOOT_AEROSOL = 2
+    INTEGER(i_kind),  PARAMETER ::  WATER_SOLUBLE_AEROSOL = 3
+    INTEGER(i_kind),  PARAMETER ::   SULFATE_CMAQ_AEROSOL = 4
+    INTEGER(i_kind),  PARAMETER ::        SEASALT_AEROSOL = 5
+    INTEGER(i_kind),  PARAMETER ::          WATER_AEROSOL = 6
+    INTEGER(i_kind),  PARAMETER ::      INSOLUBLE_AEROSOL = 7
+    INTEGER(i_kind),  PARAMETER ::      DUST_LIKE_AEROSOL = 8
+
+
+    integer(i_kind) , intent(in)    :: km                ! number of levels
+    integer(i_kind) , intent(in)    :: na                ! number of aerosols
+    integer(i_kind) , intent(in)    :: na_crtm           ! number of aerosols seen by CRTM
+    character(len=*), intent(in)    :: aero_name(na)     ! [na]    GOCART aerosol names
+    real(r_kind),     intent(inout) :: aero_conc(km,na)  ! [km,na] aerosol concentration (Kg/m2)
+    real(r_kind),     intent(in)    :: rh(km)            ! [km]    relative humidity [0,1]
+    type(CRTM_Aerosol_type), intent(inout) :: aerosol(na_crtm)!   CRTM Aerosol object
+
+    Real(r_kind),    Parameter :: def_diam( 3 )   = (/ 15.0E-3, 80.0E-3, 600.0E-3 /) !um for CRTM 
+    Real(r_kind),    Parameter :: def_sigma_g( 3 ) = (/ 1.70_r_kind, 2.0_r_kind,2.2_r_kind /)
+ 
+    integer(i_kind) :: i, k, irh
+
+    integer(i_kind) :: indx_dust1, indx_dust2, indx_dust3, indx_dust4,indx_dust5
+    integer(i_kind) :: indx_bc1, indx_oc1
+
+    indx_bc1=-1; indx_oc1=-1; indx_dust1=-1; indx_dust2=-1
+    indx_dust3=-1; indx_dust4=-1; indx_dust5=-1;
+
+!!! Attention !!! 
+!!! Please use this interface as a start point to modify.
+!!!  Users need to adapt the aerosol mapping parts for their own application! 
+    
+    do i = 1, na_crtm
+       if (crtm_aerosol_model == "CMAQ")then
+         select case ( trim(aero_name(i)) )
+           case ('aalj','acaj','afej','akj','amgj','amnj','asij','atij')
+             aerosol(i)%type  = DUST_CMAQ_AEROSOL
+           case ('asoil','acors')
+             aerosol(i)%type  = DUST_LIKE_AEROSOL
+           case ('aeci','aecj')
+             aerosol(i)%type  = SOOT_AEROSOL
+           case ('alvoo1i','alvoo2i','anh4i','anh4j','ano3i','ano3j','aso4k','asvoo1i','asvoo2i')
+             aerosol(i)%type  = WATER_SOLUBLE_AEROSOL
+           case ('aso4i','aso4j','acli','anh4k','ano3k')
+             aerosol(i)%type  = SULFATE_CMAQ_AEROSOL 
+           case ('aclj','aclk','anai','anaj','aseacat')
+             aerosol(i)%type  = SEASALT_AEROSOL
+           case ('aivpo1j','alvpo1i','alvpo1j','aothri','aothrj','asvpo1i','asvpo1j','asvpo2i','asvpo2j','asvpo3j','atol1j','axyl1j','axyl2j','axyl3j')
```

Comment:
This line exceeds the 132 character limit.  Please break this line down so that it doesn't exceed 132 characters.